### PR TITLE
change samples to saved object

### PR DESCRIPTION
**Pull request by derek-ho**
### Description
Saves sample panels as saved object instead of observability ops panel

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/components/custom_panels/helpers/utils.tsx [hunk 1]
```diff
@@ -50,6 +50,8 @@ export const isNameValid = (name: string) => {
   return name.length >= 50 || name.length === 0 ? false : true;
 };
 
+export const samplePanelName = '[Logs] Web traffic Panel';
```
Comment (ps48) [inline public/components/custom_panels/helpers/utils.tsx:53]: minor: should we move this to common/constants?

File: public/components/custom_panels/redux/panel_slice.ts [hunk 2]
```diff
@@ -21,11 +21,12 @@ import {
 } from '../../../../common/types/custom_panels';
 import { coreRefs } from '../../../framework/core_refs';
 import { SavedObject, SimpleSavedObject } from '../../../../../../src/core/public';
-import { isNameValid } from '../helpers/utils';
+import { isNameValid, samplePanelName } from '../helpers/utils';
 import {
   addMultipleVisualizations,
   addVisualizationPanel,
 } from '../helpers/add_visualization_helper';
+import { createDemoPanel } from '../../../../server/common/helpers/custom_panels/sample_panels';
```
Comment (ps48) [inline public/components/custom_panels/redux/panel_slice.ts:29]: Should we move this to `/common/constants/utils`? Cause, the `createDemoPanel` function is defined in the `server` whereas it is being used in `public` component.

**Review by ps48 — APPROVED** [review]

File: public/components/custom_panels/redux/panel_slice.ts (hunk 2 above)
Comment (pjfitzgibbons) [reply public/components/custom_panels/redux/panel_slice.ts:29]: Yes, +1, please relocate `createDemoPanel` fn

**Review by pjfitzgibbons — APPROVED** [review]

File: public/components/custom_panels/redux/panel_slice.ts (hunk 2 above)
Comment (derek-ho) [reply public/components/custom_panels/redux/panel_slice.ts:29]: It's also being used in server here: https://github.com/opensearch-project/dashboards-observability/blob/72def7dde74c2611b5144fc65603e65ec814acd5/server/adaptors/custom_panels/custom_panel_adaptor.ts#L9

**Pull request merged**